sound/soc/codecs/arizona-jack.c

```diff
@@ -238,7 +238,7 @@ static void arizona_start_mic(struct arizona_priv *info)
 	unsigned int mode;
 
 	/* Microphone detection can't use idle mode */
-	pm_runtime_get_sync(arizona->dev);
+	pm_runtime_resume_and_get(arizona->dev);
```

Comment:
Hmm. @plbossart is this safe usage when replacing get_sync where return value is not checked. After this change, usage count is not increased on error, but execution is still continued and there's an eventual unconditional call to pm_runtime_put(). I don't see __pm_runtime_suspend() in power.c handling this case gracefully.